ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -16,7 +16,33 @@
 import { InvalidSearchFormMessage } from '../../../component/announcement/CommonMessages'
 const announcement = require('../../../component/announcement/index.jsx')
 
-export function validate(errors: any) {
+export function validate(filters: any, errors: any) {
```

Comment:
I would recommend creating a new method for your new logic, so we don't break other uses of `validate`.